test/Sheet.test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-undef, no-unused-expressions */
+const { expect } = require('chai');
+// const nock = require('nock');
+const Sheet = require('../src/models/Sheet');
+const data = require('../src/mockdata');
+
+describe('Sheet model', () => {
+  const sheet = new Sheet(data.SHEET1_DATA);
+
+  it('should work', () => {
+    expect(true).to.be.ok;
+  });
+
+  it('should fetch the sheet range', () => {
```

Comment:
@meghanprestemon please leave comments for future developers where we are going to have to refactor after actually connecting to the Google APIs. (maybe even at the top of the files if this will happen in many places.